commands/generate.js

```diff
@@ -6,16 +6,18 @@ const generateFile = require('../helpers/generateFile');
 const generateStore = require('../helpers/generateStore');
 
 const name = program.args[1];
+const targetName = path.basename(name)
+const targetPath = path.dirname(name)
 let type = program.args[0];
 
-const testPath = path.join(__dirname, `../templates/${type}.test.ts`);
+const testSrc = path.join(__dirname, `../templates/${type}.test.ts`);
 const shouldBeFunctionalComponent = program.commands[0].functional;
 
 if (type === 'store') {
-  generateStore(name);
+  generateStore(targetName, targetPath);
 } else {
-  if (fs.existsSync(testPath)) {
-    generateFile(name, testPath, 'test');
+  if (fs.existsSync(testSrc)) {
+    generateFile(targetName, targetPath, testSrc, 'test');
```

Comment:
i dont like the idea of to many paremetsr per function, maybe better to pass singe object with proper values in keys? For me always 3 is the max and over that iam changing it to object.
But its minor thing, and if you are guys want to have 4 parameters there im fine with that.

Reply:
that makes sense,  I'd go with it as well

Reply:
Sure, I'll go for single argument with proper keys.